stable/mariadb/templates/master-pdb.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.master.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "mariadb.fullname" . }}
+  labels:
+    app: {{ template "mariadb.name" . | quote }}
```

Comment:
Revert this change. It doesn't work

```
Error: render error in "mariadb/templates/slave-pdb.yaml": template: mariadb/templates/_helpers.tpl:6:18: executing "mariadb.name" at <.Chart.Name>: can't evaluate field Chart in type string
```

Reply:
I tested it using the command below:

```
$ helm template --name mariadb stable/mariadb --set master.podDisruptionBudget.enabled=true,slave.podDisruptionBudget.enabled=true
```

Reply:
Oops, will revert. So can you please clarify your previous note?

> For consistency with the `metadata.labels.release`, use: `release: {{ .Release.Name | quote }}`